### ISSUE TITLE
[New Releases] Play icons don't have correct styling

### ISSUE THREAD

**Issue by Plueres** [spicetify/spicetify-cli#2008]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.97.962.g24733a46
- Spicetify version: v2.14.1
```

### 📝 Description

The play icons that show when hovering over a new release in the new releases app don't have the correct styling, or no styling at all. When hovering over a song or playlist now, it turns into a very tiny light gray rectangle with a white triangle inside.

### 📸 Screenshots

![image](https://user-images.githubusercontent.com/71077562/200818400-e7d4bc1f-623f-4c7b-b151-64e7ad0442ba.png)

**Comment by kyrie25** [comment]
Can't replicate on the latest version of Spotify/Spicetify. Try re-installing Spicetify/downloading the latest copy of New Releases.
![image](https://user-images.githubusercontent.com/77577746/201107636-5fc840c8-8b0b-4a33-91d6-5ba44cc04640.png)

**Comment by kyrie25** [comment]
This was already fixed in #1887
